File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-07-15/source/2026-07-15-reliability-runbook-alert-triage-automation.ts
// Northstar Transit Cloud | Checkout SVC-17 | 2026-07-13 | error budget 2.4% | INC-260713
export const topic = "alert_triage";
export const fingerprint = "f25ac1f0f07a52ab";
export const alertLevel = (depth) => depth >= 24000 ? 'critical' : depth >= 12000 ? 'warning' : 'normal';

File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-07-15/source/2026-07-15-reliability-runbook-alert-triage-automation.rs
// Northstar Transit Cloud | Checkout SVC-17 | 2026-07-13 | error budget 2.4% | INC-260713
pub const TOPIC: &str = "alert_triage";
pub const FINGERPRINT: &str = "4ee13ee96667f816";
pub fn q07_is_critical(depth: u32) -> bool { depth >= 24_000 }
